### PR TITLE
Release/v9.1.8

### PR DESCRIPTION
**Pull request by AliAbdelfattah**
## Summary of Changes

- Fixes an issue with iOS invocation events causing the welcome message not to show.
- Updated the pubspec file format to conform with the pub store guidelines.
- Created a new sample project and linked it to the iOS and Android CI.
- Removed the example project.
- Enabled the Android UI tests.
- Updated the CI to use XCode 12.3 and fixed the broken steps.
- Updated the .gitignore file with the latest recommended files from the flutter team.
- Fixed the XCode warnings for the miswritten function param docs in the iOS header files.

## Checklist
- [ ] In addition to adding the new API to Dart, I have also added its implementation to both iOS and Android.
- [ ] I have added at least 1 unit test in Dart for the new API and made sure the tests pass locally.
- [ ] I have updated [README.md](https://github.com/Instabug/Instabug-Flutter/blob/master/README.md) to add the new API.

### CODE REVIEW

**Review by alyezz — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## v9.1.8 (2020-10-05)
```
Comment (AliAbdelfattah) [inline CHANGELOG.md:1]: Update date

File: example/.flutter-plugins-dependencies [hunk 2]
```diff
@@ -0,0 +1 @@
+{"info":"This is a generated file; do not edit or check into version control.","plugins":{"ios":[{"name":"instabug_flutter","path":"/Users/aliabdelfattah/Documents/Dev/Instabug/Dev_Apps/Instabug-Flutter/","dependencies":[]}],"android":[{"name":"instabug_flutter","path":"/Users/aliabdelfattah/Documents/Dev/Instabug/Dev_Apps/Instabug-Flutter/","dependencies":[]}],"macos":[],"linux":[],"windows":[],"web":[]},"dependencyGraph":[{"name":"instabug_flutter","dependencies":[]}],"date_created":"2021-02-05 23:43:02.556588","version":"1.20.4"}
```
Comment (AliAbdelfattah) [inline example/.flutter-plugins-dependencies:1]: Add the file to .gitignore

File: example/android/app/src/main/java/com/instabug/instabugflutterexample/CustomFlutterApplication.java [hunk 3]
```diff
@@ -12,7 +12,7 @@ public void onCreate() {
     ArrayList<String> invocation_events = new ArrayList<>();
     invocation_events.add(InstabugFlutterPlugin.INVOCATION_EVENT_FLOATING_BUTTON);
     InstabugFlutterPlugin instabug = new InstabugFlutterPlugin();
-    instabug.start(CustomFlutterApplication.this, "2d355f559ea67051a56fce82603f8e41", invocation_events);
+    instabug.start(CustomFlutterApplication.this, "3cc65af4ae01a17a2e8edb09b32537cd", invocation_events);
```
Comment (AliAbdelfattah) [inline example/android/app/src/main/java/com/instabug/instabugflutterexample/CustomFlutterApplication.java:15]: Undo token change

File: example/ios/Flutter/.last_build_id [hunk 4]
```diff
@@ -0,0 +1 @@
+383d0aa192e5769e24f82a9d3297e0f3
```
Comment (AliAbdelfattah) [inline example/ios/Flutter/.last_build_id:1]: Add to .gitignore

File: example/ios/Flutter/flutter_export_environment.sh [hunk 5]
```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```
Comment (AliAbdelfattah) [inline example/ios/Flutter/flutter_export_environment.sh:1]: Add to .gitignore

File: example/ios/Podfile [hunk 6]
```diff
@@ -9,90 +12,29 @@ project 'Runner', {
...
 post_install do |installer|
   installer.pods_project.targets.each do |target|
-    target.build_configurations.each do |config|
-      config.build_settings['ENABLE_BITCODE'] = 'NO'
-    end
+    flutter_additional_ios_build_settings(target)
   end
 end
-
-pod 'OCMock', '~> 3.4'
```
Comment (AliAbdelfattah) [inline example/ios/Podfile:98]: Undo

File: example/ios/instabug_flutter_exampleTests/instabug_flutter_exampleTests.m [hunk 7]
```diff
@@ -7,137 +7,137 @@
 //
 
 #import <XCTest/XCTest.h>
-#import "OCMock/OCMock.h"
-#import "InstabugFlutterPlugin.h"
+// #import "OCMock/OCMock.h"
```
Comment (AliAbdelfattah) [inline example/ios/instabug_flutter_exampleTests/instabug_flutter_exampleTests.m:10]: Undo the whole file

File: example/lib/main.dart [hunk 8]
```diff
@@ -34,6 +35,7 @@ class _MyAppState extends State<MyApp> {
       Instabug.start('efa41f402620b5654f2af2b86e387029',
           <InvocationEvent>[InvocationEvent.floatingButton]);
     }
+    InstabugLog.logVerbose('VERBOOOOOOSE');
```
Comment (AliAbdelfattah) [inline example/lib/main.dart:38]: Undo

**Review by bsameh-at-instabug — APPROVED** [review]

**Pull request merged**